tv-shows-app/src/components/features/auth/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,89 @@
+'use client'
+
+import { mutatorLogin } from "@/fetchers/mutator";
+import { Alert, AlertIcon, Button, FormControl, FormLabel, Heading, Input, Link, Text, chakra } from "@chakra-ui/react";
+import { useRouter } from "next/navigation";
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+import useSWRMutation from "swr/mutation";
+import NexLink from "next/link";
+import { swrKeys } from "@/fetchers/swrKeys";
+
+interface ILoginFormInputs {
+	email: string;
+	password: string;
+}
+
+export const LoginForm = () => {
+	const router = useRouter();
+	const {register, handleSubmit} = useForm<ILoginFormInputs>();
+	const [globalError, setGlobalError] = useState([]);
+
+	const { trigger } = useSWRMutation(swrKeys.login, mutatorLogin, {
+		onSuccess: (data) => {
+			if (data.status === 'error') {
+				setGlobalError(data.data);
+				return;
+			}
+
+			router.push('/');
+		},
+	});
+
+	const onLogin = async(data: ILoginFormInputs) => {
+		setGlobalError([]);
+		await trigger(data);
+	}
+
+	return (
+		<chakra.form
+			flexDirection="column"
+			alignItems="center"
+			display="flex"
+			gap={4}
+			onSubmit={handleSubmit(onLogin)}
+		>
+			<Heading
+				as="h2"
+				size="lg"
+			>
+				Login to your
+			</Heading>
+
+			<Text
+				textAlign='center'
+			>
+				Please login to your account or register if you don't have one.
+			</Text>
+
+			{globalError.length > 0 &&
+				<Alert
+					color='black'
+					status='error'
+				>
+					<AlertIcon />
+					{globalError.map((error, index) => error)}
+				</Alert>
+			}
+
+			<FormControl isRequired={true}>
+				<FormLabel>Email</FormLabel>
+				<Input {...register("email", )} required type="email" />
```

Comment:
Validacije bi bilo super da hendlas kroz lib kao sto smo pricali:
```suggestion
				<Input {...register("email", { required: true } )} type="email" />
```